django/contrib/gis/db/models/fields.py

```diff
@@ -243,6 +243,12 @@ def deconstruct(self):
             kwargs['dim'] = self.dim
         if self.geography is not False:
             kwargs['geography'] = self.geography
+
+        # Oracle-specific attributes for `USER_SDO_GEOM_METADATA`
+        if self._extent != (-180.0, -90.0, 180.0, 90.0):
+            kwargs['extent'] = self._extent
+        if self._tolerance != 0.05:
+            kwargs['tolerance'] = self._tolerance
```

Comment:
I wonder if it's worth adding two class attributes constants for these defaults.

Reply:
Well by looking previous ones they're not class constants either (default dimensions and geography).

Reply:
Right, this was more a of suggestion open to discussion given the composite nature of `(-180.0, -90.0, 180.0, 90.0)`. I kind of wish `Field.deconstruct` was more smart wrt to `__init__` defaults by relying on `inspect` reflection in the first place.